apps/console/src/app.tsx

```diff
@@ -39,7 +44,8 @@ import { ResourceTypes } from "@wso2is/admin.template-core.v1/models/templates";
 import ExtensionTemplatesProvider from "@wso2is/admin.template-core.v1/provider/extension-templates-provider";
 import { AppConstants as CommonAppConstants } from "@wso2is/core/constants";
 import { IdentityAppsApiException } from "@wso2is/core/exceptions";
-import { CommonHelpers, isPortalAccessGranted } from "@wso2is/core/helpers";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Is this expected?

Reply:
Yes, because in order to use the useRequiredScopes hook, we need to have the app component wrapped with AccessControlProvider. Currently it is available for components below the app component, hence this suppression